tests/components/notify/test_yessssms.py

```diff
@@ -0,0 +1,124 @@
+"""The tests for the notify yessssms platform."""
+import unittest
+import requests_mock
+from homeassistant.components.notify import yessssms
+
+
+class TestNotifyYesssSMS(unittest.TestCase):
+    """Test the yessssms notify."""
+
+    def setUp(self):  # pylint: disable=invalid-name
+        """Set up things to be run when tests are started."""
+        login = "06641234567"
+        passwd = "testpasswd"
+        recipient = "06501234567"
+        self.yessssms = yessssms.YesssSMSNotificationService(login,
```

Comment:
**Preference:**
These extra lines can be reduced with this alternative wrapping syntax:
```python
self.yessssms = yessssms.YesssSMSNotificationService(
    login, passwd, recipient)
```